source/content/create-custom-upstream.md

```diff
@@ -180,6 +188,37 @@ To avoid incompatibilities, you must track Pantheon's corresponding upstream rep
 
     </Tab>
 
+    <Tab title="Drupal 9" id="d91">
+
+    1. Pull in the core files and commit them first:
+
+     ```bash{promptUser: user}
+     git checkout -b master
+     git fetch pantheon-drupal-9 master
+     git merge pantheon-drops-9/master
+     git push origin master
+     ```
+
+    1. Drupal 9 on Pantheon includes [Integrated Composer](/integrated-composer) to manage dependencies.
```

Comment:
I would almost just send them to the Integrated Composer docs so this information isn't duplicated and have to be updated in multiple places later.

If not, just be explicit that they need to `cd upstream-config` before updating the packages in the upstream.

With this composer structure, we just need to be really clear that there are TWO `composer.json` files, and which one they should be updating.

Reply:
I added a `cd upstream-config` line, and don't `cd` back, but I'm not sure that's necessarily bad, considering the rest of the doc
moving the doc to tech review and we can change it if needed. I'll leave this conversation open for now